riva/clients/nmt/riva_streaming_speech_translate_client.cc

```diff
@@ -71,6 +71,7 @@ DEFINE_bool(
     "True returns text exactly as it was said with no normalization.  False applies text inverse "
     "normalization");
 DEFINE_string(ssl_cert, "", "Path to SSL client certificatates file");
+DEFINE_string(nmt_text_file, "s2t_output.txt", "NMT text file");
```

Comment:
can you clarify in the help string that this is the output file and what will be the expected contents of this file?

Reply:
do we always want to dump to file ?

Reply:
yes let me update the help string. Yes, output file is expected to be generated for each client command run.

Reply:
S2S client already dumps the audio output file for each run, so thinking to have the same behavior for S2T client also.